---
bool success
sensor_msgs/Image background
